packages/core/lib/commands/packages.js

```diff
@@ -0,0 +1,24 @@
+var command = {
+  command: "packages",
+  description: "List available packages on connected EthPM Registry",
+  builder: {},
+  help: {
+    usage: "truffle packages",
+    options: []
+  },
+  run: async function (options, done) {
```

Comment:
(async functions shouldn't take callbacks also... one or the other, else things can break in subtle ways)